### PR TITLE
Add dev tool to add a card

### PR DESCRIPTION
**Pull request by maxammann**
* Self verify button
* option to add card in About tab
* Fixed endless loading spinner
* Fixed broken card verification

### CODE REVIEW

**Review by f1sh1918 — APPROVED** [review]
Tested on ios simulator without code review. Works fine :)

File: frontend/lib/verification/verification_qr_scanner_page.dart [hunk 1]
```diff
@@ -39,7 +44,19 @@ class VerificationQrScannerPage extends StatelessWidget {
...
+              onPressed: () async {
+                final provider = Provider.of<CardDetailsModel>(context, listen: false);
+                final cardDetails = provider.cardDetails!;
+                final generator = OTPGenerator(cardDetails.totpSecretBase32);
+                final code =
+                    encodeVerificationCardDetails(VerificationCardDetails(cardDetails, generator.generateOTP().code));
+                _handleQrCode(context, code);
+              },
+              child: const Text("Self Verify"),
+            )
```
Comment (michael-markl) [inline frontend/lib/verification/verification_qr_scanner_page.dart:59]: I think I'm not a huge fan of putting this here.
Can't we put this into the dev settings?

File: frontend/lib/about/dev_settings_view.dart [hunk 2]
```diff
@@ -69,6 +75,62 @@ class DevSettingsView extends StatelessWidget {
...
   }
 
+  Future<void> _showRawCardInput(BuildContext context) async {
+    showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        final base64Controller = TextEditingController();
+        return AlertDialog(
+          scrollable: true,
+          title: const Text('Set base64 card'),
```
Comment (michael-markl) [inline frontend/lib/about/dev_settings_view.dart:85]:
```suggestion
          title: const Text('Activate Card from Base64'),
```

File: frontend/lib/about/dev_settings_view.dart [hunk 3]
```diff
@@ -69,6 +75,62 @@ class DevSettingsView extends StatelessWidget {
...
+                      icon: Icon(Icons.card_membership),
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          ),
+          actions: [
+            TextButton(
+              child: const Text("Submit"),
```
Comment (michael-markl) [inline frontend/lib/about/dev_settings_view.dart:107]:
```suggestion
              child: const Text("Activate Card"),
```

File: frontend/lib/about/dev_settings_view.dart [hunk 4]
```diff
@@ -69,6 +75,62 @@ class DevSettingsView extends StatelessWidget {
...
+              onPressed: () {
+                final messengerState = ScaffoldMessenger.of(context);
+                final provider = Provider.of<CardDetailsModel>(context, listen: false);
+                try {
+                  IdentificationQrContentParser(provider).processQrCodeContent(base64Controller.text);
+                  messengerState.showSnackBar(
+                    const SnackBar(
+                      content: Text("Card set"),
```
Comment (michael-markl) [inline frontend/lib/about/dev_settings_view.dart:115]:
```suggestion
                      content: Text("Card activated."),
```

**Review by michael-markl — APPROVED** [review]
Only code review.

File: frontend/lib/verification/verification_qr_scanner_page.dart (hunk 1 above)
Comment (maxammann) [reply frontend/lib/verification/verification_qr_scanner_page.dart:59]: All of this will be refactored soon (hopefully soon). So I'll leave this for now.

**Pull request merged**